### DISCUSSION TITLE
Problem starting Airsim block.

### DISCUSSION THREAD

**Discussion by JohnVorwald** [microsoft/AirSim#4225] [General]
[AirSimStartupError.odt](https://github.com/microsoft/AirSim/files/7720937/AirSimStartupError.odt)
Attached file contains screen shots.  I tried running UE4Editor and loading the blocks project.  But I get the error "Engine modules are out of date, and cannot be compiled while the engine is running.  Please build through your IDE".

I ran setup.sh and build.sh in the Airsim clone directory.

**Comment by zimmy87** [comment]
Have you ran https://github.com/microsoft/AirSim/blob/master/Unreal/Environments/Blocks/update_from_git.sh after build.sh?
